### ISSUE TITLE
sputnik does not resume on stop/start

### ISSUE THREAD

**Issue by crccheck** [texastribune/docker-sputnik#3]
I thought it was only a problem with the external volume, but it looks like the simpler version has the same problem

**Comment by blackbicbiro** [comment]
did you get anywhere with this. Just having a look at it now with version 2.4

**Comment by crccheck** [comment]
nope. I don't think this is in use anymore either. It's stable enough where we just put a health check on it and just re-launched it as needed. With the external volume, you don't lose any config.

**Comment by blackbicbiro** [comment]
Seems that if i modify the start script to stop Sputnik inside the container and then start it again using "service Sputnik stop", "service Sputnik start" it start Sputnik properly. 
wierd. cant seem to get to the bottom
Thanks

**Comment by crccheck** [comment]
I guess this issue predates `docker exec`. I try to only `docker exec` for development/diagnosis though and would still just make a new image. I wonder if we're still even using this.

**Comment by blackbicbiro** [comment]
Im just about to build a server with it running as we have potentially 10 x bond2 units going live in the next coming months and need to run multiple instances on the one server. i think it has something to do with SIGTERM not shutting down the server properly when i do a docker stop. if it times out after 10 secons it does a SIGKILL. will play and see what i can do

Cheers
